Jenkinsfile_servlet

```diff
@@ -28,44 +27,81 @@ node('linux'){
               globalMavenSettingsConfig: 'oss-settings.xml',
               mavenOpts: '-Xms2g -Xmx4g',
               mavenLocalRepo: ".repository") {
-        sh "mvn -V -B clean install -DskipTests -T10 -e"
+        sh "mvn -V -B -pl jetty-home -am clean install -DskipTests -T2 -e"
       }
     }
   }
   stage("Setup servlet tck"){
-    env.JAVA_HOME='${tool "$jdk"}'
-    env.PATH="${env.JAVA_HOME}/bin:${env.PATH}"
-    sh "wget https://jenkins.webtide.net/userContent/servlettck-4.0_latest.zip"
-    sh "unzip servlettck-4.0_latest.zip"
+    echo "Starting withEnv()"
+    withEnv(["JAVA_HOME=${ tool "$jdk" }", "PATH=${ tool "$jdk" }/bin:${env.PATH}"]) {
+      // env.JAVA_HOME='${tool "$jdk"}'
+      // env.PATH="${env.JAVA_HOME}/bin:${env.PATH}"
+
+      echo "JAVA_HOME=${env.JAVA_HOME}"
+      echo "PATH=${env.PATH}"
+
+      sh "ls -la ${env.JAVA_HOME}"
+
+      // Execute some simple lookups of required command line applications
+      // If the application isn't found, then it fails the build
+      sh "which curl"
+      sh "which wget"
+      sh "which unzip"
+      sh "which find"
+      sh "which java"
+
+      echo "Fetching servlettck from ${env.JENKINS_URL}userContent/tcks/servlettck-4.0_latest.zip"
+      sh "curl -O ${env.JENKINS_URL}userContent/tcks/servlettck-4.0_latest.zip"
+
+      echo "Unpacking Servlet TCK"
+      sh "unzip servlettck-4.0_latest.zip"
+
+      sh "cd jetty-home/target/ && mkdir jetty-base"
 
-    sh "cd jetty-distribution/target/ && mkdir jetty-base"
-    sh "cd jetty-distribution/target/jetty-base && java -jar ../distribution/start.jar --create-startd --add-to-start=resources,server,http,https,http2c,webapp,deploy,jsp,jmx,jmx-remote"
-    sh 'find servlettck -name *.war -exec cp {} jetty-distribution/target/jetty-base/webapps/ \\;'
+      sh "ls -la jetty-home/target/"
+      sh "ls -la jetty-home/target/jetty-base"
 
-    unstash name: 'ts.jte'
-    // replace values in ts.jte
-    def text = readFile "ts.jte"
-    text = text.replaceAll("@WORKSPACE@", "${env.WORKSPACE}")
-    writeFile file:"servlettck/bin/ts.jte", text: text
+      echo "Running home to create startd"
 
-    unstash name: 'realm.ini'
-    sh "cp realm.ini jetty-distribution/target/jetty-base/start.d"
+      sh "cd jetty-home/target/jetty-base && java -jar ../jetty-home/start.jar --approve-all-licenses --create-startd --add-to-start=resources,server,http,https,http2c,webapp,deploy,jsp,jmx,jmx-remote"
```

Comment:
Added `--approve-all-licenses` for safety.
Also, made it use `jetty-home` instead of jetty-distribution